content/documentation/pages/9-applications/4-application-metadata.md

```diff
@@ -85,19 +85,22 @@ Depending on the runtime, the metadata is packaged either as an separate compani
 
 ```xml
 <plugin>
- 	<groupId>org.springframework.cloud</groupId>
- 	<artifactId>spring-cloud-app-starter-metadata-maven-plugin</artifactId>
-    <version>2.0.0.RELEASE</version>
- 	<executions>
- 		<execution>
- 			<id>aggregate-metadata</id>
- 			<phase>compile</phase>
- 			<goals>
- 				<goal>aggregate-metadata</goal>
- 			</goals>
- 		</execution>
- 	</executions>
- </plugin>
+   <groupId>org.springframework.cloud</groupId>
+   <artifactId>spring-cloud-dataflow-apps-metadata-plugin</artifactId>
+   <version>1.0.2</version>
+   <configuration>
+      <storeFilteredMetadata>true</storeFilteredMetadata>
+   </configuration>
+   <executions>
+      <execution>
+         <id>aggregate-metadata</id>
+         <phase>compile</phase>
+         <goals>
+            <goal>aggregate-metadata</goal>
+         </goals>
+      </execution>
+   </executions>
+</plugin>
```

Comment:
For the mojo plugin we did not have the `<id>` property    Do we want to remove it from the example?
Here is my cut of the plugin configuration:
```
			<plugin>
				<groupId>org.codehaus.mojo</groupId>
				<artifactId>properties-maven-plugin</artifactId>
				<version>1.0.0</version>
				<executions>
					<execution>
						<phase>process-classes</phase>
						<goals>
							<goal>read-project-properties</goal>
						</goals>
						<configuration>
							<files>
								<file>
									${project.build.outputDirectory}/META-INF/spring-configuration-metadata-encoded.properties
								</file>
							</files>
						</configuration>
					</execution>
				</executions>
			</plugin>
```

Reply:
Sure @cppwfs go ahead and remove it.